SBAT.md

```diff
@@ -213,68 +214,95 @@ product. Setting a product-specific generation number for such an event
 eliminates the need for other vendors to have to re-release the binaries for
 their products with an incremented global number.
 
-However, once the global number is bumped for the next upstream CVE fix there
-will be no further need to carry that product-specific generation number.
-Satisfying the check of the global number will also exclude any of the older
-product-specific binaries.
+The current consensus is for both generation numbers to only ever go up and not
+be reset.
 
-For example: There is a global CVE disclosure and all vendors coordinate to
-release fixed components on the disclosure date.  This release bumps the global
-generation number for GRUB to 4.
+#### Example: a vendor forking a global project
 
-SBAT revocation data would then require a GRUB with a global generation number
-of 4.
+Let's imagine a fictional company named "Vendor C" having an active fork of the
+upstream GNU GRUB2. Therefore, Vendor C provides their own product-specific
+generation number. This is happening at the point in time, when the upstream
+product's entry starts with `grub,3`, hence why Vendor C's product ships with
+entries similar to:
+
+```
+sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
+grub,3,Free Software Foundation,[...]
+grub.vendorc,1,Vendor C,[...]
+```
+
+Suddenly there is a global CVE disclosure and all vendors coordinate
+to release fixed components on the disclosure date. This release bumps the
+global generation number for GRUB from 3 to 4. Vendor C's product's binaries
+are now shipped with the entries:
+
+```
+sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
+grub,4,Free Software Foundation,[...]
+grub.vendorc,1,Vendor C,[...]
+```
+
+In the perfect scenario, to provide the perfect security, the UEFI SBAT
+revocation variable (named *SbatLevel*) would then be set, so that GRUB with a
+global generation number of only 4 or higher would be able to be booted. (In
+reality there's naturally going to be a certain grace period)
```

Comment:
I don't think you need to emphasise the "perfect security" etc. so much here. Maybe just:

After this, the UEFI SBAT revocation variable (named *SbatLevel*) would be updated to raise the minimum global generation number for GRUB to 4.

?